### DISCUSSION TITLE
eslint 报错

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [nshen/learn-neovim-lua#81] [Q&A]
![image](https://user-images.githubusercontent.com/77786952/183831022-ee8852ba-1f93-4e87-9346-60fc680064e9.png)

React 前端开发 windows wsl。
每一个页面顶部都有找不到 node 的报错
我的node 是通过 nvm 控制版本的

**Comment by Linya-IronMan** [comment]
/usr/bin/env 是一个看不懂的乱码文件

**Reply by nshen** [reply]
在wsl里能 node -v 吗

**Reply by Linya-IronMan** [reply]
可以
